bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -28,18 +28,18 @@ def __init__(self):
 
 class _gROOTWrapper(object):
     """Internal class to manage lookups of gROOT in the facade.
```

Comment:
NFC again.

Reply:
It's part of the first formatting-only commit